andela/lotus-ah-frontend
# #160451570 Users should be able to follow each other

#### What does this PR do?
- Implement follow functionality

#### Description of Task to be completed?
- create a list following page
- create a list followers page 
- update profile page to show following and followers
- follow button on view article page should follow and unfollow an author as the case may be

#### How should this be manually tested?
- start the app
- register two users
- create an article with the first user account
- view the article with the second user account click follow
- the first user should receive a notification, of a new follower
- go to your profile by clicking on the username on the header
    you should see you following and followers count
- click on view on the button for the followers or following 
- you should see a corresponding list page

#### Any background context you want to provide?
NA

#### What are the relevant pivotal tracker stories?
[#160451570](https://www.pivotaltracker.com/story/show/160451570, "user follow functionlaity")

#### Screenshots (if appropriate)
<img width="1680" alt="screen shot 2018-11-27 at 12 56 19 am" src="https://user-images.githubusercontent.com/24890605/49049591-ea8e5e80-f1df-11e8-9326-981783487a30.png">
<img width="1680" alt="screen shot 2018-11-27 at 12 57 48 am" src="https://user-images.githubusercontent.com/24890605/49049592-ea8e5e80-f1df-11e8-89a9-aafd04064902.png">
<img width="1680" alt="screen shot 2018-11-27 at 12 58 05 am" src="https://user-images.githubusercontent.com/24890605/49049599-f1b56c80-f1df-11e8-8369-82cb12986afd.png">
<img width="1680" alt="screen shot 2018-11-27 at 12 58 26 am" src="https://user-images.githubusercontent.com/24890605/49049604-f8dc7a80-f1df-11e8-92fb-e075c2376d6c.png">



#### Questions:
